plugin-annotation/src/test/java/com/mapbox/maps/plugin/annotation/generated/CircleAnnotationManagerTest.kt

```diff
@@ -102,12 +105,12 @@ class CircleAnnotationManagerTest {
     }
     every {
       mapFeatureQueryDelegate.queryRenderedFeatures(
-        any<ScreenCoordinate>(),
-        any(),
+        any<RenderedQueryGeometry>(), any(),
```

Comment:
Seems these tests are generated, how do I regenerate them instead of manual change @tobrun ?

Reply:
Ok, found it, regenerated with make from internal repo